### DISCUSSION TITLE
ZbOccupancy automatic time out problem

### DISCUSSION THREAD

**Discussion by Sayed-afridy** [arendst/Tasmota#19614] [Q&A]
I am using tasmota on sonoff zbbridge. the problem is it auto generates the occupancy zero for my sonoff zigbee pir sensor even if the sensor still detects movement. I want it to get occupancy zero when the sensor sends zero

12:21:45.442 RSL: RESULT = {"Rule1":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":300,"Free":211,"Rules":"on ZbReceived#0xB637#Power=1 do publish cmnd/backyard/Power5 TOGGLE endon on ZbReceived#0xB637#Power=0 do publish cmnd/backyard/Power2 %value% endon on ZbReceived#0xB637#Power=2 do publish cmnd/backyard/POWER1 TOGGLE endon on ZbReceived#0x49E3#Occupancy do WebSend [192.168.0.108] POWER %value% endon"}}
12:22:14.709 RSL: SENSOR = {"ZbReceived":{"0x49E3":{"Device":"0x49E3","0500?00":"010000000000","ZoneStatusChange":1,"Occupancy":1,"Endpoint":1,"LinkQuality":48}}}
12:22:14.722 RUL: ZBRECEIVED#0X49E3#OCCUPANCY performs "WebSend [192.168.0.108] POWER 1"
12:22:14.984 RSL: RESULT = {"WebSend":"Done"}
12:23:03.042 MQT: Attempting connection...
12:23:03.256 MQT: Connect failed to 192.168.0.103:1883, rc -2. Retry in 90 sec
12:23:44.734 RSL: SENSOR = {"ZbReceived":{"0x49E3":{"Device":"0x49E3","Occupancy":0}}}
12:23:44.748 RUL: ZBRECEIVED#0X49E3#OCCUPANCY performs "WebSend [192.168.0.108] POWER 0"
12:23:44.961 RSL: RESULT = {"WebSend":"Done"}

**Comment by s-hadinger** [comment]
As per documentation, do `ZbOccupancy 0x49E3,0`
